addons/stock/wizard/stock_zero_quantity_count.py

```diff
@@ -0,0 +1,46 @@
+#  -*- coding: utf-8 -*-
+#  Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+
+from odoo import models, fields, api
+
+
+class StockZeroQuantityCount(models.TransientModel):
+
+    _name = 'stock.zero.quantity.count'
+    _description = 'Zero Quantity Count'
+
+    location_ids = fields.Many2many('stock.location')
+    location_names = fields.Char('Locations Names', compute='_compute_location_names')
```

Comment:
the other pre action done wizard all ahve a pickign_ids field which contains the pickings impacted by the wizard